include/vapor/Grid.h

```diff
@@ -1195,6 +1199,8 @@ class VDF_API Grid {
         }
     }
 
+    double               _defaultZ = 0.0;
```

Comment:
There is already a convention for the default Z and it is not always zero so this may break some things.

Reply:
This PR is still in draft status, but feedback is welcome. Can you elaborate on the existing convention?

Reply:
John requested my review. Check out Renderer::GetDefaultZ

Reply:
I see. Let me check it out.

Reply:
Sorry. I didn't see that it was marked draft. We can pause the review.

Reply:
There's no defaultZ in grids anymore.